packages/eslint-plugin-udemy/rules/import-blacklist/index.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const blacklist = [
```

Comment:
👍

Reply:
Maybe you can provide default configuration that they can embrace and extend or replace entirely if they want to.

Reply:
I feel like the current blacklist can't be a sane default though since it's quite unique to website-django/static use case (e.g. especially the exception paths and the set of libraries used together).

Reply:
Thanks! I've moved the blacklist to a context option.